nimble/host/include/host/ble_gap.h

```diff
@@ -1379,6 +1424,64 @@ int ble_gap_periodic_adv_create_sync_cancel(void);
  */
 int ble_gap_periodic_adv_terminate_sync(uint16_t sync_handle);
 
+#if MYNEWT_VAL(BLE_PERIODIC_ADV_SYNC_TRANSFER)
+/**
+ * Disable or enable periodic reports for specified sync.
+ *
+ * @param sync_handle        Handle identifying synchronization.
+ * @param enable             If reports should be enabled.
+ *
+ * @return                   0 on success; nonzero on failure.
+ */
+int ble_gap_periodic_adv_receive_enable(uint16_t sync_handle, bool enable);
```

Comment:
This function can be mistaken for a function used to receive sync transfer. I think the name should include "report".